js/main.js

```diff
@@ -14,6 +14,17 @@ function rit(position) {
     var mylocation = "lat: " + position.coords.latitude +"<br>";
     mylocation += "lng: " + position.coords.longitude+"<br><br>";
     output.innerHTML = mylocation;
+    my_preview = "https://www.google.com/maps/?q=" + position.coords.latitude + "," +position.coords.longitude;
+    preview.innerHTML = "<a target='_blank' href='"+my_preview+"'>PREVIEW on GMaps!</a>"+"<br><br>";
```

Comment:
Why we need extra link? User can click on the marker and we can rediret to the Google Maps